src/hough.rs

```diff
@@ -55,14 +55,11 @@ pub fn detect_lines(image: &GrayImage, options: LineDetectionOptions) -> Vec<Pol
             let p = unsafe { image.unsafe_get_pixel(x, y)[0] };
 
             if p > 0 {
-                for m in 0..180u32 {
+                for (m, (c, s)) in cos_lut.iter().zip(&sin_lut).enumerate() {
```

Comment:
It could be faster and clearer to simply merge the two vectors into one vector `Vec<(f32, f32)>` now that I think about it.